shared_model/validators/transaction_validator.hpp

```diff
@@ -227,6 +228,23 @@ namespace shared_model {
         return reason;
       }
 
+      ReasonsGroupType operator()(
+          const interface::CompareAndSetAccountDetail &casad) const {
```

Comment:
`casad` :confused: 
but I see, you follow the above naming scheme (though ugly to my taste)

Reply:
It is the naming scheme, maybe is csad better?